### PR TITLE
Added active admin resources and `paranoia` to User model.

### PR DESCRIPTION
**Pull request by Zawgor24**

### CODE REVIEW

File: app/admin/articles.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+ActiveAdmin.register Article do
+  permit_params :avatar, :body, :title, :user_id
+
+  form do |f|
+    f.inputs 'Required information' do
+      f.input :avatar
+      f.input :body
+      f.input :title
+      f.input :user_id, label: 'Member', as: :select,
```
Comment (IlyaUmanets) [inline app/admin/articles.rb]: Use I18n please for `"Member"`

File: app/admin/posts.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ActiveAdmin.register Post do
+  permit_params :avatar, :body, :category_id, :title, :user_id
+
+  form do |f|
+    f.inputs 'Required information' do
+      f.input :avatar
+      f.input :body
+      f.input :category
+      f.input :title
+      f.input :user_id, label: 'Member', as: :select,
```
Comment (IlyaUmanets) [inline app/admin/posts.rb]: same

File: app/admin/users.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register User do
+  permit_params :address, :avatar, :birthday, :email, :first_name, :info,
+    :last_name, :password, :sex
+
+  active_admin_paranoia
+
+  form do |f|
+    f.inputs 'Required information' do
```
Comment (IlyaUmanets) [inline app/admin/users.rb]: Try use I18n please here too.

File: app/views/layouts/application.html.slim [hunk 4]
```diff
@@ -15,17 +15,8 @@ html
         #nav-content.collapse.navbar-collapse
           ul.navbar-nav
             li.nav-item
-              a.nav-link href='/users/4'  Profile
+              a.nav-link href="/users/#{current_user.id}"  Profile
```
Comment (IlyaUmanets) [inline app/views/layouts/application.html.slim]: Use `link_to` helper

File: app/views/layouts/application.html.slim [hunk 5]
```diff
@@ -15,17 +15,8 @@ html
...
-              button#dropdownMenuButton.btn.btn-secondary.dropdown-toggle aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" type="button" 
-                | Dropdown button
-              .dropdown-menu aria-labelledby="dropdownMenuButton" 
-                a.dropdown-item href="#"  Action
-                a.dropdown-item href="#"  Another action
-                a.dropdown-item href="#"  Something else here
-
-
-            button.logout.btn.btn-danger.navbar-btn= link_to 'Log out', destroy_user_session_path, method: :delete
+              a.nav-link href='/users/#{current_user.id}/edit'  Edit profile
```
Comment (IlyaUmanets) [inline app/views/layouts/application.html.slim]: Same

File: db/seeds.rb [hunk 6]
```diff
@@ -21,3 +16,8 @@
                       user_id: news1.user_id)
 news2.comments.create(title: 'second', body: 'comment',
                       user_id: news2.user_id)
+
+if Rails.env.development?
```
Comment (IlyaUmanets) [inline db/seeds.rb]: No need to check development

**Review by IlyaUmanets — CHANGES_REQUESTED** [review]

File: app/admin/articles.rb (hunk 1 above)
Comment (Zawgor24) [reply app/admin/articles.rb]: Done.

File: app/admin/posts.rb (hunk 2 above)
Comment (Zawgor24) [reply app/admin/posts.rb]: OK, done.

File: app/views/layouts/application.html.slim (hunk 4 above)
Comment (Zawgor24) [reply app/views/layouts/application.html.slim]: Ok, replaced by link_to.

File: app/views/layouts/application.html.slim (hunk 5 above)
Comment (Zawgor24) [reply app/views/layouts/application.html.slim]: Done.

File: db/seeds.rb (hunk 6 above)
Comment (Zawgor24) [reply db/seeds.rb]: Got it.

**Pull request merged**